src/util/DeviceFingerprint.js

```diff
@@ -16,6 +16,12 @@ define(['vendor/lib/q', 'okta/jquery'], function (Q, $) {
 
   return {
     generateDeviceFingerprint: function (oktaDomainUrl, element) {
+      if (!navigator.userAgent) {
+        return Q.reject('user agent is not defined');
+      } else if (isWindowsPhone()) {
+        return Q.reject('device fingerprint is not supported in Windows phones');
+      }
```

Comment:
This sounds like a feature change. Are we okay with not sending ``X-Device-Fingerprint`` for Windows phones? It seems that doing this won't allow us to capture new Windows phone devices.
Also nit: "device fingerprint is not supported **on** Windows phones".